ml/misc/planning.md

```diff
@@ -0,0 +1,82 @@
+# Interview Timeline
+
+> Agile-like stories for interview preparation.
+> Interview data: 4/22/2019(est.). Start data: 3/24/2019
+> 27 days
+
+
+
+## Content
+
+
```

Comment:
* No need extra blank lines under the heading. One blank line should be enough. 
* Be consistent of using blank lines